lib/portlayer/exec/container_cache.go

```diff
@@ -79,3 +117,109 @@ func (conCache *containerCache) Remove(idOrRef string) {
 	}
 
 }
+
+func isContainerID(id string) bool {
+	// ID should only be lower case chars & numbers
+	match, _ := regexp.Match("^[a-z0-9]*$", []byte(id))
+	return match
+}
+
+// call back for the event stream
+func callme(ie event.Event, sess *session.Session) {
```

Comment:
I suggest calling this callback unless there is another function maybe and you want to do :)

```
defer callme(maybe(...))
```